.travis.yml

```diff
@@ -0,0 +1,25 @@
+jdk:
+  - oraclejdk8
```

Comment:
I wonder what jdk8 version travis is using. For that matter, what jdk8 are we using in dev/alpha/staging/prod? It would be good to ensure that consistent versions are used everywhere (there are bugfixes, etc between JDK minor versions).